src/models/Scholarships.ts

```diff
@@ -4,10 +4,9 @@ import FirestoreCollection from './base/FirestoreCollection';
 import FirestoreModelList from './base/FiretoreModelList';
 import ScholarshipEligibility from '../types/ScholarshipEligibility';
 import FirestoreModel from './base/FirestoreModel';
+import Model from './base/Model';
 
-interface ScholarshipData {
-  // TODO(https://github.com/beyondhb1079/s4us/issues/56):
-  // Update this to reflect the schema
+export interface ScholarshipData {
```

Comment:
Does this need to be exported?

Reply:
oops, was testing something and forgot to change it back.